.github/workflows/pull_request_template.md

```diff
@@ -0,0 +1,3 @@
+Checklist
+
+- [ ] Does not contain AI generated text or code.
```

Comment:
@ccoVeille I've left this here, so that we can build it further. This will be the template for future pull requests.

You can later add the CI controls, asking people to confirm that they pass the test